src/css/_finding-card.scss

```diff
@@ -1,12 +1,13 @@
 .finding-card-wrapper {
-  display: block;
-  position: relative;
-  padding: $space-xxs;
-  padding-top: $space-xs;
-  padding-left: $space-l;
+  display: flex;
+  align-items: center;
+  height: 25px;
   color: white;
+  padding-left: 22px;
+  margin: 0 2px;
   border-radius: 5px;
-  margin: 1px 3px;
+  cursor: pointer;
```

Comment:
The only think I use cursor: pointer for is links (underlined, blue links that point to webpages like docs). Most everything in the App is `cursor: default`.

This is different from most web apps out there which make everything that is possibly clickable `pointer: cursor`. However, native apps tend not to. Since we're all into being native, let's mimic them.

https://medium.com/simple-human/buttons-shouldnt-have-a-hand-cursor-b11e99ca374b

Reply:
This also applies to the Section Header UI elements. All those can be cursor: default as well.